charts/camunda-platform-8.5/values.yaml

```diff
@@ -173,6 +173,8 @@ global:
       protocol: https
       host:
       port: 443
+    ## @param global.opensearch.prefix Opensearch.prefix defines the prefix which is used by the Zeebe Opensearch Exporter to create Opensearch indexes
```

Comment:
No need for `Opensearch.prefix`.
It should be `global.opensearch.prefix defines ...`

Reply:
Also, let's stick to the official name `OpenSearch` (capital O and capital S, not `Opensearch`).